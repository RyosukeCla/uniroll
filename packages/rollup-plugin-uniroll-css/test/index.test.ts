// @ts-ignore
import precss from "precss";
import postcss from "postcss";
import autoprefixer from "autoprefixer";
import nano from "cssnano";

test("minify", async () => {
  const prefix = `/* autoprefixer grid: autoplace */\n`;
  const result = await postcss([
    precss,
    autoprefixer({
      overrideBrowserslist: ["last 2 versions", "ie 11"],
    }),
    nano,
  ]).process(prefix + "div { display: grid; }", { from: undefined });
  expect(result.css).toMatchSnapshot();
});
