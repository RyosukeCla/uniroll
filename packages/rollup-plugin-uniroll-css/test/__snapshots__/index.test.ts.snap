// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`minify 1`] = `"div{display:-ms-grid;display:grid}"`;
